### DISCUSSION TITLE
git pulls timeout

### DISCUSSION THREAD

**Discussion by jpalus** [community/community#69775] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Doing ie `:PackerSync` ([packer.nvim](https://github.com/wbthomason/packer.nvim)) from within `neovim` that checks for plugin updates by doing `git pull` started to result in timeouts. Each time different repository set timeouts and update proces never completes fully. The error message is ie:
```console
fatal: unable to access 'https://github.com/neovim/nvim-lspconfig.git/': Failed to connect to github.com port 443 after 76703 ms: Couldn't connect to server
```
Same behavior on two different networks

**Comment by ebubekirgungor** [comment]
Helle, you can try to turn off ssl verification
`git config --global http.sslverify "false"`
If does not work, do not forget turn on back with `git config --global http.sslverify "true"`

**Reply by jpalus** [reply]
Still same timeouts. I don't think it's SSL related though. The error message indicates like new TCP connection cannot be opened so much before SSL exchange can be made over that connection.

**Reply by ebubekirgungor** [reply]
Could it be related to clone timeout? You can try "clone_timeout = false" in your packer config file.

**Reply by jpalus** [reply]
The behavior is exactly the same if I do `git pull` on all of pakcer's repositories in command line. As a matter of fact error in description is from such attempt. And as error indicates it gives up after `76703 ms`. I'm surely not gonna go higher than that.

**Reply by ebubekirgungor** [reply]
Probably it can not connect because of last "/" in the url "https://github.com/neovim/nvim-lspconfig.git/"
Can you try to change the config like that:
`default_url_format = 'git@github.com:%s'`

**Reply by jpalus** [reply]
When I wrote _"Each time different repository set timeouts"_ I really meant that. One time `nvim-lspconfig`times out, and the other try it succeeds. No it's not that too. How can it be? Location is communicated once connection is established. Connection fails to establish though.

**Reply by ebubekirgungor** [reply]
I found an issue about this: https://github.com/wbthomason/packer.nvim/issues/1019 It says this problem fixed with:
`git = { default_url_format = 'git@github.com:%s' },`
The error shows the url is 'https://github.com/neovim/nvim-lspconfig.git/'. Somehow it should be without last "/".

**Comment by jpalus** [comment]
Looks like connection issue is gone today and multiple updates in a row fully succeed again.
